File: sleet-api-service/src/main/java/com/sleet/api/model/Token.java
package com.sleet.api.model;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonProperty;

/**
 * Represents a response from the TD API for authentication purposes
 *
 * @author mautomic
 */
@JsonIgnoreProperties(ignoreUnknown = true)
public class Token {

    @JsonProperty("access_token")
    public String accessToken;

    @JsonProperty("refresh_token")
    public String refreshToken;

    public String getAccessToken() {
        return accessToken;
    }

    public String getRefreshToken() {
        return refreshToken;
    }
}

File: sleet-api-service/src/main/java/com/sleet/api/model/Token.kt
package com.sleet.api.model

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import com.fasterxml.jackson.annotation.JsonProperty

/**
 * Represents a response from the TD API for authentication purposes
 *
 * @author mautomic
 */
@JsonIgnoreProperties(ignoreUnknown = true)
open class Token {
    @JsonProperty("access_token")
    var accessToken: String? = null
    @JsonProperty("refresh_token")
    var refreshToken: String? = null
}